code/addons/test/src/components/TestDiscrepancyMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Link } from 'storybook/internal/components';
+import { useStorybookApi } from 'storybook/internal/manager-api';
+import { styled } from 'storybook/internal/theming';
+
+import { CallStates } from '@storybook/instrumenter';
+
+import { DOCUMENTATION_LINK } from '../constants';
+
+const Wrapper = styled.div(
+  ({ theme: { color, typography, background } }) => `
+  text-align: start;
+  padding: 11px 15px;
+  font-size: ${typography.size.s2}px;
+  font-weight: ${typography.weight.regular};
+  line-height: 1rem;
+  background: ${background.app};
+  border-bottom: 1px solid ${color.border};
+  color: ${color.defaultText};
+  background-clip: padding-box;
+  position: relative;
+  code {
+    font-size: ${typography.size.s1 - 1}px;
+    color: inherit;
+    margin: 0 0.2em;
+    padding: 0 0.2em;
+    background: rgba(255, 255, 255, 0.8);
+    border-radius: 2px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1);
+  }
+`
+);
+
+interface TestDiscrepancyMessageProps {
+  browserTestStatus: CallStates;
+}
+export const TestDiscrepancyMessage = ({ browserTestStatus }: TestDiscrepancyMessageProps) => {
+  const api = useStorybookApi();
+  const docsUrl = api.getDocsUrl({
+    subpath: DOCUMENTATION_LINK,
+    versioned: true,
+    renderer: true,
+  });
+  const message = `This component test passed in ${browserTestStatus === CallStates.DONE ? 'this browser' : 'CLI'}, but the tests failed in ${browserTestStatus === CallStates.ERROR ? 'this browser' : 'CLI'}.`;
```

Comment:
I am not a wording expert, but the message is too vague. What does the difference between "this browser" and "CLI" mean in detail? It means that there are two different ways of running tests, and they differ in their output. Can we make this somehow clearer? WDYT?

Reply:
I think it's easier to make it clear in the troubleshooting page. Unless we can provide a short message that is clearer than that? cc @MichaelArestad

Reply:
@valentinpalkovic Do you have any suggestions to make it clearer? Like Yann mentioned, I was thinking it would be better to walk through in detail in docs.

Reply:
Ok. Then lets leave it as is. I don't have a better suggestion right now.